tools/serverpod_cli/lib/src/analyzer/entities/validation/validate_node.dart

```diff
@@ -11,10 +10,10 @@ class ValidateNode {
 
   /// If set, the key must match the restriction, add any errors to the collector.
```

Comment:
Chore: This comment and the ones below are outdated now.

Please also check if there are other mentions of the collector that now should be changed or removed, for example in places that used to take in a collector.